dynawo/sources/Models/Modelica/PreassembledModels/CMakeListsModelsTargets.txt

```diff
@@ -0,0 +1,26 @@
+# Define utility targets
+
+## Generators
+add_custom_target(GENERATOR_MODELS)
+
+file(GLOB Generators Generator*.xml)
+
+foreach(generator IN LISTS Generators)
```

Comment:
Yes we can and should. This PR was just a proposition to have a way to add this helper targets later on... I can either do it now or in a complementary PR. The Dynaflow target was added by Marco because the use is very imminent and the use to only compile SMIB models is a current need for Modelica models developpers.

Reply:
yes, I take care of DynaSwing one